src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf

```diff
@@ -234,12 +234,12 @@
       </trans-unit>
       <trans-unit id="ERR_DoesNotOverrideMethodFromObject">
         <source>'{0}' does not override the method from 'object'.</source>
-        <target state="new">'{0}' does not override the method from 'object'.</target>
+        <target state="translated">“{0}”不会通过 "object" 覆盖方法。</target>
```

Comment:
```suggestion
        <target state="translated">“{0}”不会覆盖 "object" 的方法。</target>
```

Reply:
we are working on review of these suggestions